internal/local/starter.go

```diff
@@ -64,11 +70,41 @@ func (s *Starter) Start(ctx context.Context, cfg *config.GKEConfig, replicaSet s
 		return nil
 	}
 
+	var caCert *x509.Certificate
+	if cfg.UseMtls {
+		// Make sure the CA certificate has been created.
+		s.once.Do(func() {
+			s.caCert, s.caKey, s.caCertErr = generateCACert()
+		})
+		if s.caCertErr != nil {
+			return s.caCertErr
+		}
+		caCert = s.caCert
+	}
+
 	// Create and run one babysitter per colocation group replica.
 	babysitters := make([]bsitter, defaultReplication)
+	create := func(ctx context.Context) (*babysitter.Babysitter, error) {
```

Comment:
This creates one cert and key for every babysitter? If the babysitters are sharing the same identity, can they share the same cert and key? Or that's bad security practice?

Reply:
According to our security contact, creating different key pairs, even with the same name, is the way to go. Sharing private keys is a no-no.